runtime/doc/options.txt

```diff
@@ -4245,6 +4245,20 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Defines the maximum time in msec between two mouse clicks for the
 	second click to be recognized as a multi click.
 
+							*'msgfunc'* *'msgfu'*
+'msgfunc' 'msgfu'	string	(default: empty)
+			global
+	This option specifies a function to be called by |messages|.
+	It overwrites the default messages behavior.
+	See |message-function| for an explanation of how to write the
+	function and an example.
+	Note: It disables 'showmode' feature.
+	Note: It is disabled if your neovim GUI enables "ext_messages"
```

Comment:
neovim -> Neovim

alternatively you could remove the word and just leave the "GUI"

Reply:
Actually it should be Nvim